### PR TITLE
Plume mainnet config

### PR DESCRIPTION
**Pull request by stgLamps**
**In this PR**

- Adding USDC configs for Plume network
- one TODO, filling in Executor address

### CODE REVIEW

File: packages/stg-evm-v2/devtools/config/mainnet/01/usdc-token.config.ts [hunk 1]
```diff
@@ -25,13 +26,16 @@ import {
...
-// Except for Peaq where it's deployed externally
+// Except for chains where it's deployed externally
 const usdcPeaqAsset = getAssetNetworkConfig(EndpointId.PEAQ_V2_MAINNET, TokenName.USDC)
 assert(usdcPeaqAsset.address != null, `External USDC address not found for PEAQ`)
 
 const usdcDegenAsset = getAssetNetworkConfig(EndpointId.DEGEN_V2_MAINNET, TokenName.USDC)
 assert(usdcDegenAsset.address != null, `External USDC address not found for DEGEN`)
 
+const usdcPlumeAsset = getAssetNetworkConfig(EndpointId.PLUME_V2_MAINNET, TokenName.USDC)
+assert(usdcPlumeAsset.address != null, `External USDC address not found for PLUME`)
```
Comment (ravinagill15) [inline packages/stg-evm-v2/devtools/config/mainnet/01/usdc-token.config.ts:37]: nice!

File: packages/stg-evm-v2/devtools/config/mainnet/01/asset.usdc.config.ts
Comment (ravinagill15) [inline packages/stg-evm-v2/devtools/config/mainnet/01/asset.usdc.config.ts:1]: Just confirming weth is not needed?

**Review by ravinagill15 — COMMENTED** [review]
Looks good! Let's wait to merge until builds pass. I think we also need a changeset?

File: packages/stg-evm-v2/devtools/config/mainnet/01/asset.usdc.config.ts
Comment (stgLamps) [reply packages/stg-evm-v2/devtools/config/mainnet/01/asset.usdc.config.ts:1]: only USDC for Plume for now!

File: packages/stg-definitions-v2/src/constant.ts [hunk 2]
```diff
@@ -1283,6 +1291,33 @@ export const NETWORKS: NetworksConfig = {
...
+                multiSendAddress: '0x1E999A606771577f10AFee9111b6263cCF176a1D',
+                multiSendCallOnlyAddress: '0x3BE538f8E6a8809892a1a9618bda69C518a14a8a',
+                safeMasterCopyAddress: '0xB79F7E683E365E21B150812b86f1e0D60512D168',
+                safeProxyFactoryAddress: '0x73aFaC05744f1246Ac94d5Ee92d56be5a7298ecC',
+                fallbackHandlerAddress: '0x1B01E51e3ff9D83DD245D8320c5007b6a3dBF91c',
+                createCallAddress: '0xF83E45c8575b7317686d4bbeE489bB6a93E6C4E3',
+                signMessageLibAddress: '0x34A62CbF94d5dF170CA67C4ECB335453117d4515',
+                simulateTxAccessorAddress: '0x9D59A44Ae2BF3A5A9751463BD4EC77b172eFa9B0',
+            },
+        },
```
Comment (ravinagill15) [inline packages/stg-definitions-v2/src/constant.ts:1319]: Need to encapsulate this within `safeConfig`

```suggestion
        safeConfig: {
            safeAddress: '0x30b1Ca299577f70bc456Fa37C6822d9088069d31',
            safeUrl: 'https://plume-tx.lzdevnet.org/',
            contractNetworks: {
                [98865]: {
                    multiSendAddress: '0x1E999A606771577f10AFee9111b6263cCF176a1D',
                    multiSendCallOnlyAddress: '0x3BE538f8E6a8809892a1a9618bda69C518a14a8a',
                    safeMasterCopyAddress: '0xB79F7E683E365E21B150812b86f1e0D60512D168',
                    safeProxyFactoryAddress: '0x73aFaC05744f1246Ac94d5Ee92d56be5a7298ecC',
                    fallbackHandlerAddress: '0x1B01E51e3ff9D83DD245D8320c5007b6a3dBF91c',
                    createCallAddress: '0xF83E45c8575b7317686d4bbeE489bB6a93E6C4E3',
                    signMessageLibAddress: '0x34A62CbF94d5dF170CA67C4ECB335453117d4515',
                    simulateTxAccessorAddress: '0x9D59A44Ae2BF3A5A9751463BD4EC77b172eFa9B0',
                },
            },
        },
```

**Review by ravinagill15 — CHANGES_REQUESTED** [review]

**Review by ravinagill15 — APPROVED** [review]

**Pull request merged**